package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "opentelemetry-base",
+  "version": "0.0.1",
+  "description": "OpenTelemetry is a distributed tracing and stats collection framework.",
+  "main": "build/src/index.js",
+  "types": "build/src/index.d.ts",
+  "scripts": {
+    "fix": "lerna run fix",
+    "postinstall": "npm run bootstrap",
+    "compile": "lerna run compile",
+    "test": "lerna run test",
+    "bootstrap": "lerna bootstrap",
+    "bump": "lerna publish",
+    "codecov": "lerna run codecov",
+    "check": "lerna run check"
+  },
+  "repository": "open-telemetry/opentelemetry-node",
+  "keywords": [
+    "opentelemetry",
+    "nodejs",
+    "tracing",
+    "profiling",
+    "metrics",
+    "stats"
+  ],
+  "author": "OpenTelemetry Authors",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "lerna": "^3.13.4",
+    "typescript": "^3.4.5"
```

Comment:
Was there ever a discussion to determine whether it makes sense to use TypeScript? As discussed in a few other issues already, I feel that for a Node library, and especially this type of project, it can actually make it very difficult to have for example different code paths per version of Node, or to test constructs that don't exist in every versions.

I think it would be a good exercise to actually weigh the pros and cons and not simply go with TypeScript because of its popularity. Also, do we have a plan for how watch mode would work with TypeScript?

Reply:
TypeScript was agreed upon by the majority. A guideline will be created to ensure it is used in a way that is compatible with this type of project.